clarity_ext/result_file.py

```diff
@@ -0,0 +1,14 @@
+class ResultFile:
```

Comment:
Shouldn't this class be placed in the domain module? It's a very small class to be contained in one module.